Invoke-CMApplyDriverPackage.ps1

```diff
@@ -1287,16 +1288,21 @@ Process {
 			else {
 				$DetectionMethodsCount = 3
 			}
-			Write-CMLogEntry -Value "[DriverPackage:$($DriverPackageItem.PackageID)]: Processing driver package with $($DetectionMethodsCount) detection methods: $($DriverPackageItem.Name)" -Severity 1
+			Write-CMLogEntry -Value "[DriverPackage:$($DriverPackageDetails.PackageID)]: Processing driver package with $($DetectionMethodsCount) detection methods: $($DriverPackageDetails.PackageName)" -Severity 1
 
 			switch ($ComputerDetectionMethod) {
 				"SystemSKU" {
-					# Attempt to match against SystemSKU
-					$ComputerDetectionMethodResult = Confirm-SystemSKU -DriverPackageInput $DriverPackageDetails.SystemSKU -ComputerData $ComputerData -ErrorAction Stop
-					
-					# Fall back to using computer model as the detection method instead of SystemSKU
-					if ($ComputerDetectionMethodResult.Detected -eq $false) {
-						$ComputerDetectionMethodResult = Confirm-ComputerModel -DriverPackageInput $DriverPackageDetails.Model -ComputerData $ComputerData
+					if ([string]::IsNullOrEmpty($DriverPackageDetails.SystemSKU)) {
+						Write-CMLogEntry -Value "[DriverPackage:$($DriverPackageDetails.PackageID)]: Driver package was skipped due to missing SystemSKU values in description field" -Severity 2
+					}
+					else {
+						# Attempt to match against SystemSKU
+						$ComputerDetectionMethodResult = Confirm-SystemSKU -DriverPackageInput $DriverPackageDetails.SystemSKU -ComputerData $ComputerData -ErrorAction Stop
+						
+						# Fall back to using computer model as the detection method instead of SystemSKU
+						if ($ComputerDetectionMethodResult.Detected -eq $false) {
+							$ComputerDetectionMethodResult = Confirm-ComputerModel -DriverPackageInput $DriverPackageDetails.Model -ComputerData $ComputerData
+						}
```

Comment:
Changes above are not mine, they are my failed attempt at getting @NickolajA  latest commit into my repository because I'm a Git n00b...